src/eval_framework/llm/aleph_alpha.py

```diff
@@ -55,6 +55,8 @@ def __init__(
         request_timeout_seconds: int = 30 * 60 + 5,
         queue_full_timeout_seconds: int = 30 * 60 + 5,
         bytes_per_token: float | None = None,
+        aa_token: str | None = os.getenv("AA_TOKEN", "dummy"),
+        aa_inference_endpoint: str | None = os.getenv("AA_INFERENCE_ENDPOINT", "dummy_endpoint"),
```

Comment:
nitpick, otherwise thanks for acting so fast on it
```suggestion
        token: str = os.getenv("AA_TOKEN", "dummy"),
        base_url: str = os.getenv("AA_INFERENCE_ENDPOINT", "dummy_endpoint"),
```